packages/model-viewer/src/model-viewer-base.ts

```diff
@@ -109,6 +110,21 @@ export interface RendererInterface {
  */
 export default class ModelViewerElementBase extends UpdatingElement {
   protected static[$template]: HTMLTemplateElement|void;
+  private static _powerPreference: string = 'high-performance';
+
+  /** @export */
+  static set powerPreference(value: string) {
+    if (ModelViewerElementBase._powerPreference == value) {
+      return;
+    }
+    ModelViewerElementBase._powerPreference = value;
+    Renderer.resetSingleton();
```

Comment:
There's some kind of bug here, but I think it's likely a long-standing issue with resetSingleton, since that's not a well-tested method. There are now errors, the but the render is just blank afterward. Let me see if I can debug (since I actually have two GPUs).

Reply:
This is not applicable anymore after your commit I assume